bin/run.py

```diff
@@ -513,6 +514,7 @@ def test(self):
                         stdout=None,
                         stderr=None,
                         timeout=self.problem.settings.timeout,
+                        memory=self.problem.limits.memory,
```

Comment:
We don't run `bt test` with a time limit, so I think it makes sense to keep the undefined memory limit as well, instead of taking the problem's memory limit?

(also for the interactive case at line 628)